### PR TITLE
Add types and decorators to imports in integration tests

### PR DESCRIPTION
**Pull request by khairalanam**
This PR fixes missing imports of types and decorators in many of the integration tests.

### CODE REVIEW

**Comment by czgdp1807** [comment]
You need to execute `python run_tests.py -u` for updating reference tests and commit the changes.

**Comment by khairalanam** [comment]
> 

image

I tried running the run_tests.py but I am getting errors.

File: tests/reference/asr-bindc_02-5092d8e.stderr [hunk 1]
```diff
@@ -1,8 +1,2 @@
-semantic error: Type mismatch in procedure call; the types must be compatible
- --> tests/errors/bindc_02.py:9:24
-  |
-9 |     s: Struct = Struct(int32obj)
-  |                        ^^^^^^^^ type mismatch (passed argument type is i32 but required type is CPtr)
-  |
-5 |     cptr_member: CPtr
-  |                  ^^^^ type mismatch (passed argument type is i32 but required type is CPtr)
+'lpython' is not recognized as an internal or external command,
+operable program or batch file.
```
Comment (Shaikh-Ubaid) [inline tests/reference/asr-bindc_02-5092d8e.stderr:2]: @khairalanam please see above. I think the tests are not properly updated.

Comment (khairalanam) [reply tests/reference/asr-bindc_02-5092d8e.stderr:2]: What is the expected output of these reference tests? I couldn't find any errors in the terminal.

image

Comment (Shaikh-Ubaid) [reply tests/reference/asr-bindc_02-5092d8e.stderr:2]: >What is the expected output of these reference tests?

The expected output of reference tests is as stored in the main branch.

> I couldn't find any errors in the terminal.

I think its a bug in the `run_tests.py -u` that it runs (updates tests) even if it cannot find `lpython`.

Comment (czgdp1807) [reply tests/reference/asr-bindc_02-5092d8e.stderr:2]: Its a Windows specific problem. Try to ask about it on Zulip.

**Pull request closed without merging**